docs/shader-modules/project.md

```diff
@@ -54,8 +54,8 @@ Uniform names are not considered part of the official API of shader modules and
 | project_uCoordinateSystem | float | COORDINATE_SYSTEM enum |
 | project_uCenter | float | coordinate origin in world space |
 | project_uScale | float | Web Mercator scale (2^zoom) |
-| project_uPixelsPerMeter | vec3 | Web Mercator pixels per meter near the current viewport center |
-| project_uPixelsPerDegree | vec3 | Web Mercator pixels per degree near the current viewport center |
+| project_uDistancePerMeter | vec3 | Web Mercator pixels per meter near the current viewport center |
```

Comment:
I totally agree with the value of changing this uniform, but to me this is really `project_uShaderWorkingCoordinateSystemUnitsPer...`.

Especially as "distance" is a concept we talk about in the user facing (coordinate system) doc, I would tend to avoid using it unqualified for an internal value.

Reply:
Admittedly "distance" is not a good name either. We also have `project_uPixelsPerUnit` right now which converts user coordinate system units (meters in METER_OFFSET or degrees in LNGLAT_OFFSET) to Web Mercator pixels. So naming world space units "unit" does not really help. :(